src/Models/CamelCaseModel.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace Saritasa\Database\Eloquent\Models;
+
+use Illuminate\Database\Eloquent\Model;
+
+/**
+ * Class CamelCaseModel
```

Comment:
Remove this line `Class CamelCaseModel` from comment. It just duplicates class declaration, so it is useless.